lib/companies_web/controllers/job_controller.ex

```diff
@@ -5,19 +5,19 @@ defmodule CompaniesWeb.JobController do
   plug :load_companies when action in [:new, :edit, :create, :update]
 
   def index(conn, _params) do
-    jobs = Jobs.list_jobs()
+    jobs = Jobs.all()
     render(conn, "index.html", jobs: jobs)
   end
 
   def new(conn, %{"company_id" => company_id}) do
-    changeset = Jobs.change_job(%Job{company_id: company_id})
-    company = Companies.get_company!(company_id)
+    changeset = Jobs.change(%Job{company_id: company_id})
+    company = Companies.get!(company_id)
     render(conn, "new.html", changeset: changeset, company: company)
   end
 
   def create(conn, %{"job" => params}) do
     case Jobs.create(params, current_user(conn)) do
-      {:ok, _job} ->
+      {:ok} ->
```

Comment:
On successful creation, what gets returned is {:ok, %PendingChange{}}, which makes this fail to pattern match.